api/v2.0/legacy_swagger.yaml

```diff
@@ -2,12 +2,12 @@ swagger: '2.0'
 info:
   title: Harbor API
   description: These APIs provide services for manipulating Harbor project.
-  version: 1.10.0
+  version: '2.0'
```

Comment:
If this version number changed to 2. Where is the swagger for 1.0? Still in this file or move to somewhere?

Reply:
There is no 1.0 anymore, all API version is v2.0 now